src/components/Button/Button.md

```diff
@@ -533,6 +533,12 @@ Link button:
 <Button text="I am a link" href="https://www.yammer.com" />
 ```
 
+Link button with newWindow:
+
+```js { "props": { "data-description": "link" } }
```

Comment:
this needs a `different data-description` from the previous link, as-is the visual dff for this one overwrote the last one. (you'll also need to approve and commit the new diff image via Docker)

Reply:
Cool, will update!